src/components/about/About.jsx

```diff
@@ -1,38 +1,46 @@
 import React from 'react';
 
+const ABOUT_SECTION_STYLE = {
+    marginBottom: '8px',
+};
+
 // TODO: adjust indentation
 const About = () => (
     <div id="about-container">
-        <div id="about-header">
-            About
+        <div className="section-headers">
+            ABOUT
         </div>
         <div className="about-section">
-            <div className="about-subheader">
+            <div
+                className="about-subheader"
+                style={ABOUT_SECTION_STYLE}
+            >
                 About Kubiquity
             </div>
             <p className="about-text">
                 Kubiquity is a Kubernetes error logging and tracking tool developed and launched in 2021. Designed for production-level applications using Kubernetes to orchestrate containers, Kubiquity provides metrics on all pods' memory use to help developers prevent OOM kill errors; additionally, Kubiquity logs Kubernetes events to highlight any errors and warnings that originate within monitored clusters.
             </p>
         </div>
         <div className="about-section">
-            <div className="about-subheader">
+            <div
+                className="about-subheader"
+                style={ABOUT_SECTION_STYLE}
+            >
+
```

Comment:
Looks like we accidentally included an extra line. Could you please delete line 29?

Reply:
Line 29 removed.